cyclus_input/predicting_the_past.xml

```diff
@@ -0,0 +1,183 @@
+        </in_commods>
+        <out_commods>
+          <val>uf6</val>
+        </out_commods>
+        <residence_time>1</residence_time>
+      </Storage>
+    </config>
+  </facility>
+  <facility>
+    <name>Enrichment1</name>
+    <config>
+      <Enrichment>
+        <feed_commod>uf6</feed_commod>
+        <feed_recipe>uf6</feed_recipe>
+        <product_commod>enr_u</product_commod>
+        <tails_commod>tails</tails_commod>
+        <max_enrich>0.045</max_enrich>
+      </Enrichment>
+    </config>
+  </facility>
+  <facility>
+    <name>FuelFab1</name>
+    <config>
+      <Storage>
+        <in_commods>
+          <val>enr_u</val>
+        </in_commods>
+        <out_commods>
+          <val>fresh_uox</val>
+        </out_commods>
+        <residence_time>1</residence_time>
+      </Storage>
+    </config>
+  </facility>
+  <facility>
+    <name>WetStorage1</name>
+    <config>
+      <Storage>
+        <in_commods>
+          <val>spent_uox</val>
+        </in_commods>
+        <out_commods>
+          <val>cool_spent_uox</val>
+        </out_commods>
+        <residence_time>72</residence_time>
+      </Storage>
+    </config>
+  </facility>
+  <facility>
+    <name>DryStorage1</name>
+    <config>
+      <Storage>
+        <in_commods>
+          <val>cool_spent_uox</val>
+        </in_commods>
+        <out_commods>
+          <val>casked_spent_uox</val>
+        </out_commods>
+      </Storage>
+    </config>
+  </facility>
+  <region>
+    <name>US</name>
+    <config>
+      <NullRegion/>
+    </config>
+    <institution>
+      <name>FuelCycle</name>
+      <initialfacilitylist>
+        <entry>
+          <prototype>Mine1</prototype>
+          <number>1</number>
+        </entry>
+        <entry>
+          <prototype>Mill1</prototype>
+          <number>1</number>
+        </entry>
+        <entry>
+          <prototype>Conversion1</prototype>
+          <number>1</number>
+        </entry>
+        <entry>
+          <prototype>Enrichment1</prototype>
+          <number>1</number>
+        </entry>
+        <entry>
+          <prototype>FuelFab1</prototype>
+          <number>1</number>
+        </entry>
+        <entry>
+          <prototype>WetStorage1</prototype>
+          <number>1</number>
+        </entry>
+        <entry>
+          <prototype>DryStorage1</prototype>
+          <number>1</number>
+        </entry>
+      </initialfacilitylist>
+      <config>
+        <ManagerInst>
+          <prototypes>
+            <val>Mine1</val>
+            <val>Mill1</val>
+            <val>Conversion1</val>
+            <val>Enrichment1</val>
+            <val>FuelFab1</val>
+            <val>WetStorage1</val>
+            <val>DryStorage1</val>
+          </prototypes>
+        </ManagerInst>
+      </config>
+    </institution>
+    <institution>
+      <name>Power</name>
+      <config>
+        <DeployInst>
+          <xi:include href="./buildtimes/buildtimes.xml#xpointer(/simulation/DeployInsts/child::*)"/>
```

Comment:
Why all these extra tags? The simulation tag is confusing because it makes the reader of the buildtimes.xml file imagine that buildtimes.xml might be a full simulation file (it's not). Similarly, DeployInsts isn't a valid tag. Finally, the things contained in the DeployInsts tag in the buildtimes.xml file are not DeployInsts. They are facilities and deployment times. Please get rid of these unnecessary and confusing tags. everywhere they appear.